src/app/business/parent.service.ts

```diff
@@ -32,6 +59,14 @@ class ParentService  {
     findById (_id: string, callback: (error: any, result: IParent) => void) {
         this._parentRepository.findById(_id, callback);
     }
+
+    isEmpty(obj) {
```

Comment:
Create one utility.ts file
and put all common logic part there in one class 
e.g 
class Utility{
static isEmpty(obj){
for(var key in obj) {
            if(obj.hasOwnProperty(key))
                return false;
        }
        return true;
}


---------------------
call 
in any file
using class name of that util file
e.g.
here
parent.service.ts file

Utility.isEmpty(res) line no.16